cromwell_tools/cromwell_tools.py

```diff
@@ -166,6 +167,7 @@ def start_workflow(
     files = {
         'workflowSource': wdl_file,
         'workflowInputs': inputs_file,
+        'workflowOnHold': json.dumps(on_hold)
```

Comment:
I never know we could use json.dumps to convert a bool. 👍

Reply:
Huh, that _is_ cool. 👍

Reply:
Yeah! I tried passing it into the `json` parameter separately so we would not have to convert the boolean to a string, but unfortunately it seems like the `json` parameter gets ignored when using the `files` parameter.